src/cts/src/Clock.cpp

```diff
@@ -95,7 +95,7 @@ Box<double> Clock::computeSinkRegionClustered(
   double yMin = std::numeric_limits<double>::max();
   double yMax = std::numeric_limits<double>::lowest();
 
-  for (const std::pair<double, double>& sinkLocation : sinks) {
+  for (const std::pair<double, double> sinkLocation : sinks) {
```

Comment:
could this be `const auto& [sink_x, sink_y]` instead of a `std::pair`?

Reply:
I think you may want just `const auto [sink_x, sink_y]` to avoid references to primitive types?